femwell/mesh/mesh.py

```diff
@@ -359,6 +359,17 @@ def mesh_from_OrderedDict(
                 "SurfacesList",
                 meshtracker.get_gmsh_xy_surfaces_from_label(label),
             )
+
+            gmsh.model.mesh.field.setNumbers(
+                n + 1,
+                "CurvesList",
+                meshtracker.get_gmsh_xy_lines_from_label(label),
+            )
+
+            gmsh.option.setNumber("Mesh.MeshSizeExtendFromBoundary", 0)
+            gmsh.option.setNumber("Mesh.MeshSizeFromPoints", 0)
+            gmsh.option.setNumber("Mesh.MeshSizeFromCurvature", 0)
```

Comment:
@simbilod didn't you put those options already somewhere? Or is that only in meshwell?

Reply:
@duarte-jfs is there a reason why you add the last three lines again?

it's already done in (and the two following lines) https://github.com/HelgeGehring/femwell/blob/9ce5de4b173bf53d0e018c0d0f4ab4ba97e13201/femwell/mesh/mesh.py#L409

Reply:
Hi, sorry for getting back to you so late. I just looked at this again. To be honest, I don't understand it either. But the behaviour is different if you incliude the lines as I did. No clue why

Reply:
Ok, then I'm totally fine with just merging it :)

Reply:
Yes this will also add the mesh constraint on the curves and not just the surface (interior)

Meshwell has much more advanced resolution settings now, we should finish the migration #52